cla_frontend/settings/base.py

```diff
@@ -267,8 +275,11 @@ def get_env_value(var_name):
         environment=os.environ.get("CLA_ENV", "unknown"),
     )
 SENTRY_PUBLIC_DSN = os.environ.get("SENTRY_PUBLIC_DSN", "")
-SOCKETIO_SERVER_URL = os.environ.get("SOCKETIO_SERVER_URL", "http://localhost:8005/socket.io")
 SITE_HOSTNAME = os.environ.get("SITE_HOSTNAME", "localhost")
+# socket.io accessible over the internet, ie for browsers
+SOCKETIO_SERVER_URL = os.environ.get("SOCKETIO_SERVER_URL", "http://localhost:8005/socket.io")
+# socket.io accessible within the cluster, for health checks
+SOCKETIO_SERVICE_URL = os.environ.get("SOCKETIO_SERVICE_URL", "localhost:8005/socket.io/")
```

Comment:
Where is this used on the cla_frontend? I think only backend makes use of this

Reply:
Smoketest